test/files/neg/badtok-1-212.scala

```diff
@@ -1,4 +1,4 @@
-// scalac: -Xsource:2.12 -deprecation -Xfatal-warnings
+// scalac: -deprecation -Xfatal-warnings
```

Comment:
This test is for 2.12 messaging; the other similar test is for 2.13.

Reply:
Ok, I've deleted the redundant test.